tests/serializers/models/base.py

```diff
@@ -108,7 +108,7 @@ def __str__(self):
         raise NotImplementedError("Not so simple")
 
     def to_string(self):
-        return "%s" % self.title
+        return str(self.title)
```

Comment:
`title` is a `CharField` so `str()` is unnecessary.

Reply:
While `TeamField` which uses `Team` is not instantiated with `null=True` below, the same applies.